upgrade/UPGRADE-v9.1.0-dev.md

```diff
@@ -31,3 +31,5 @@ There you can find links to upgrade notes for other versions too.
 - add phpstan-symfony extension ([#1961](https://github.com/shopsys/shopsys/pull/1961))
     - see #project-base-diff to update your project
 
+- fix phpstan tests path ([#1974](https://github.com/shopsys/shopsys/pull/1974))
```

Comment:
IMO the upgrade notes are not necessary as the line in phpstan.neon was added recetly in master branch without release yet